### ISSUE TITLE
Support for bosonic systems

### ISSUE THREAD

**Issue by zxm403089989** [SpM-lab/Nevanlinna.jl#7]
The doc shows some examples on Fermionic models. I'm wondering that whether it is possible to see some example codes for boson Green's function? I suppose they may be helpful.

**Comment by kfkq** [comment]
Agree, though there are paper about it J. Phys. Soc. Jpn. 92, 035001 (2023), but I am not sure the details of implementation here.
